src/reducers/index.ts

```diff
@@ -16,7 +16,10 @@ var initialState = {
   displayOptions: {
     rootTypeId: null,
     skipRelay: true,
-    sortByAlphabet: false
+    sortByAlphabet: false,
+    hideDocs: false,
+    hideRoot: false,
+    transformSchema: null,
```

Comment:
@timsuchanek I think we should pass this through React components props and trigger some action when it changes so redux knows about this.
Would you prefer to do it yourself or I can merge your PR and do it on top of your work?